pandas/tests/generic/test_frame.py

```diff
@@ -245,8 +247,12 @@ class TestToXArray:
         and LooseVersion(xarray.__version__) < LooseVersion("0.10.0"),
         reason="xarray >= 0.10.0 required",
     )
-    @pytest.mark.parametrize("index", tm.all_index_generator(3))
-    def test_to_xarray_index_types(self, index):
+    def test_to_xarray_index_types(self, indices):
+        if isinstance(indices, ABCMultiIndex):
+            pytest.skip("MultiIndex is tested separately")
```

Comment:
MultiIndex is [tested right below](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/generic/test_frame.py#L308-L316)